src/getOffsetRect.ts

```diff
@@ -28,7 +28,7 @@ export function getOffsetRect(el: HTMLElement | Text | null): Bounds {
   return {
     top: Math.round(box.top + scrollTop - clientTop),
     left: Math.round(box.left + scrollLeft - clientLeft),
-    width: el.offsetWidth,
-    height: el.offsetHeight,
+    width: box.width,
+    height: box.height,
```

Comment:
> **Note**
> `offsetWidth` / `offsetHeight` округляет значение до целого числа. Если вам нужно дробное значение, используйте [element.getBoundingClientRect()](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect).

Необходимо округлить значение

```suggestion
    width: Math.round(box.width),
    height: Math.round(box.height),
```